build.yaml

```diff
@@ -221,7 +224,8 @@ common_cli:
     export FLOCKER_FUNCTIONAL_TEST_OPENSTACK_REGION=dfw
 
   setup_coverage: &setup_coverage |
-    parse_logs pip install coverage==3.7.1 http://data.hybridcluster.net/python/coverage_reporter-0.01_hl0-py27-none-any.whl  --trusted-host \$TRUSTED_HOST
+    parse_logs pip install coverage  \
+      -i \$PIP_INDEX_URL --trusted-host \$TRUSTED_HOST
 
```

Comment:
The package from  data.hybridcluster.com is `coverage_reporter`, which it doesn't appear that jenkins is using. It is a package designed to report added lines that aren't covered (as a small attempt at getting coverage changes introduced in the PR).